src/app/api/companies/[companyId]/messages/templates/bulk/route.ts

```
@@ -0,0 +1,277 @@
+      }, { status: 403 });
+    }
+
+    // Perform the bulk action
+    let result: any = {};
+    
+    switch (action) {
+      case 'activate':
+        const { error: activateError } = await supabase
+          .from('message_templates')
+          .update({ is_active: true })
+          .in('id', template_ids);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)